x-pack/plugins/observability_solution/entity_manager/server/saved_objects/entity_definition.ts

```diff
@@ -18,6 +18,7 @@ export const entityDefinition: SavedObjectsType = {
     dynamic: false,
     properties: {
       id: { type: 'keyword' },
+      version: { type: 'keyword' },
```

Comment:
Did you manually run the script to update `packages/kbn-check-mappings-update-cli/current_fields.json` ? I don't see a model version for this mapping addition?

Reply:
Oh, yes. Was that wrong?
I just saw the CI message which said to run the script with `--override` and `--no-verify` flags 🙈

Reply:
As discussed on slack - a model version with a `mappings_addition` change reflecting those additions must be added.